[32mINFO    [0m common.recordlog:conftest.py:21 -------------接口测试开始--------------
[32mINFO    [0m common.recordlog:sendrequest.py:142 接口名称：搜索用户
[32mINFO    [0m common.recordlog:sendrequest.py:143 请求地址：http://47.113.145.175:8080/ssm_web/user/findAllUserByPage
[32mINFO    [0m common.recordlog:sendrequest.py:144 请求方式：post
[32mINFO    [0m common.recordlog:sendrequest.py:145 测试用例名称：5.分页查询-第二页
[32mINFO    [0m common.recordlog:sendrequest.py:146 请求头：{'Content-Type': 'application/json', 'Authorization': 'e30bb109-1bcb-4762-914d-7010d331cc01'}
[32mINFO    [0m common.recordlog:sendrequest.py:154 请求参数：{'json': {'currentPage': 2, 'pageSize': 10, 'username': '15588886234'}}
[32mINFO    [0m common.recordlog:sendrequest.py:108 接口返回信息：{"success":true,"state":200,"message":"响应成功","content":{"pageNum":2,"pageSize":10,"size":0,"orderBy":null,"startRow":0,"endRow":0,"total":1,"pages":1,"list":[],"firstPage":1,"prePage":1,"nextPage":0,"lastPage":1,"isFirstPage":false,"isLastPage":false,"hasPreviousPage":true,"hasNextPage":false,"navigatePages":8,"navigatepageNums":[1]}}
[32mINFO    [0m common.recordlog:assertions.py:308 yaml文件预期结果：[{'contains': {'message': '响应成功'}}, {'contains': {'status_code': 200}}, {'contains': {'success': True}}, {'contains': {'content.pageNum': 2}}, {'contains': {'content.pageSize': 10}}, {'contains': {'content.isFirstPage': False}}, {'contains': {'content.hasPreviousPage': True}}, {'contains': {'content.list[0].name': '15588886234'}}]
[32mINFO    [0m common.recordlog:assertions.py:65 响应文本断言成功：预期结果【响应成功】,实际结果【响应成功】
[1m[31mERROR   [0m common.recordlog:assertions.py:41 接口返回码包含断言成功：预期结果【200】,实际结果【200】
[32mINFO    [0m common.recordlog:assertions.py:65 响应文本断言成功：预期结果【True】,实际结果【[True]】
[32mINFO    [0m common.recordlog:assertions.py:65 响应文本断言成功：预期结果【2】,实际结果【[2]】
[32mINFO    [0m common.recordlog:assertions.py:65 响应文本断言成功：预期结果【10】,实际结果【[10]】
[32mINFO    [0m common.recordlog:assertions.py:65 响应文本断言成功：预期结果【False】,实际结果【[False]】
[32mINFO    [0m common.recordlog:assertions.py:65 响应文本断言成功：预期结果【True】,实际结果【[True]】
[1m[31mERROR   [0m common.recordlog:assertions.py:50 响应键缺失断言失败：未找到【content.list[0].name】
[1m[31mERROR   [0m common.recordlog:assertions.py:342 测试失败
[32mINFO    [0m common.recordlog:conftest.py:23 -------------接口测试结束--------------